gnovm/tests/files/panic0b.gno

```diff
@@ -14,6 +14,29 @@ func f() {
 	panic("first")
 }
 
+// Stacktrace:
+// panic first
+// panic((const ("first" <untyped> string)))
+//     main/files/panic0b.gno:14
+// f<VPBlock(3,1)>()
+//     main/files/panic0b.gno:4
+// main()
+//     main/files/panic0b.gno:3
```

Comment:
I was wondering why the defer function wasn't showing up in this stacktrace, so I wound up on a 10-minute adventure that ended up being a full-day adventure: https://github.com/gnolang/gno/pull/2597

Reply:
It's beautiful 😍, I thought it was designed like this

Reply:
It does leave out information, say if I call `defer f.Close()`, and a panic occurs in `Close()`, then the stacktrace doesn't show the additional frame of the `Close` function.